hugo/data/teams.yml

```diff
@@ -94,6 +76,21 @@ sections:
           name: Denys Smirnov
           position: Senior Software Engineer
           github: https://github.com/dennwc
+
+      - name: Data Processing Team
+        members:
+        - picture: antonio.png
+          name: Antonio Navarro
+          position: Lead Software Engineer
+          github: https://github.com/ajnavarro          
+        - picture: kuba.png
+          name: Kuba Podgórski
+          position: Senior Software Engineer
+          github: https://github.com/kuba--
+        - picture: miguel.png
+          name: Miguel Molina
+          position: Senior Software Engineer
+          github: https://github.com/erizocosmico
         - picture: juanjo.png
```

Comment:
Given the ordering rules Juanjo should be between Antonio and Kuba

Reply:
Thanks! Done.